packages/suite/src/components/suite/WebUsbButton.tsx

```diff
@@ -1,30 +1,54 @@
 import TrezorConnect from '@trezor/connect';
-import { ButtonProps, Button } from '@trezor/components';
+import { ButtonProps, Button, IconButton, Tooltip } from '@trezor/components';
 import { Translation, TranslationKey } from './Translation';
 
 interface WebUsbButtonProps extends Omit<ButtonProps, 'children' | 'icon'> {
     translationId?: TranslationKey;
     icon?: ButtonProps['icon'] | false;
 }
 
+const handleClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.stopPropagation();
+    TrezorConnect.requestWebUSBDevice();
+};
+
 export const WebUsbButton = ({
     translationId = 'TR_CHECK_FOR_DEVICES',
     icon = 'SEARCH',
     size = 'tiny',
+    variant = 'primary',
     ...rest
 }: WebUsbButtonProps) => (
     <div data-test="web-usb-button">
         <Button
             {...rest}
             icon={icon === false ? undefined : icon}
             size={size}
-            variant="primary"
-            onClick={e => {
-                e.stopPropagation();
-                TrezorConnect.requestWebUSBDevice();
-            }}
+            variant={variant}
+            onClick={handleClick}
         >
             <Translation id={translationId} />
         </Button>
     </div>
 );
+
+export const WebUsbIconButton = ({
+    translationId = 'TR_CHECK_FOR_DEVICES',
+    size = 'tiny',
+    variant = 'primary',
+    ...rest
+}: WebUsbButtonProps) => {
```

Comment:
nit: inline function

Reply:
I created a proposal for this: https://github.com/trezor/trezor-suite/issues/13075